### PR TITLE
Set name to containers

### PR DESCRIPTION
**Pull request by stdevMac**
## Changes:
- Added flag `--container-tag` to add a suffix to sedge containers name.

## Types of changes
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Review by AntiD2ta — COMMENTED** [review]
Although this PR needs to be reviewed mostly by @adriantpaez. I have a main concern with the name convention you choose that is not the same as we agreed. Need that to be discussed before proceeding to review the entire PR

File: internal/pkg/services/services.go [hunk 1]
```diff
@@ -21,9 +21,11 @@ import (
 )
 
 const (
-	ServiceCtSlashingData    = "validator-slashing-data"
-	ServiceCtValidator       = "validator-client"
-	ServiceCtValidatorImport = "validator-import-client"
+	ServiceCtSlashingData       = "validator-slashing-data"
+	ServiceCtValidatorImport    = "validator-import-client"
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply internal/pkg/services/services.go]: Was a mistake, already fixed it

[the remark this answers is not in the document]
Comment (adriantpaez) [reply internal/pkg/services/services.go]: `ServiceCtSlashingData` still without the `sedge` prefix

**Review by adriantpaez — APPROVED** [review]

File: cli/actions/generation_test.go [hunk 2]
```diff
@@ -73,8 +74,8 @@ func TestGenerateDockerCompose(t *testing.T) {
 
 	// Check that the execution service is set.
 	assert.NotNil(t, composeData.Services.Execution)
-	// Check that the execution container name is `execution-client'.
-	assert.Equal(t, composeData.Services.Execution.ContainerName, "execution-client")
+	// Check that the execution container name is `sedge-execution-client_sampleTag'.
```
Comment (AntiD2ta) [inline cli/actions/generation_test.go:77]:
```suggestion
	// Check that the execution container name is `sedge-execution-client-sampleTag'.
```

File: internal/pkg/generate/types.go [hunk 3]
```diff
@@ -75,6 +75,7 @@ type GenData struct {
 	ExecutionApiUrl         string
 	ExecutionAuthUrl        string
 	ConsensusApiUrl         string
+	ContainerTags           string
```
Comment (AntiD2ta) [inline internal/pkg/generate/types.go:78]: Change to `ContainerTag` please, is only one tag. Isn't it?

File: cli/actions/generation_test.go (hunk 2 above)
Comment (AntiD2ta) [reply cli/actions/generation_test.go:77]: If there isn't any tag you should test if the suffix `-sampleTag` is not there

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Minor changes requested. Is almost there! Good work

**Review by AntiD2ta — APPROVED** [review]
Approved!

**Pull request merged**